src/components/notebook.js

```diff
@@ -2,14 +2,30 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Iframe from 'react-iframe'
 
-import Typography from '@material-ui/core/Typography';
+import { Typography, withStyles } from '@material-ui/core';
 
 import PageHead from './page-head';
 import PageBody from './page-body';
 
-export default class Notebook extends Component {
+const styles = () => ({
+  iframe: {
+    border: 0,
+    left: 0,
+    position: 'absolute',
+    top: 0,
+    width:'100%',
+    height:'100%'
+  },
+  container: {
+    overflow: 'hidden',
+    position: 'relative',
+    paddingTop: '72%'
```

Comment:
Let give ourselves a little more height and increase this to 100%

Reply:
Fixed.